### DISCUSSION TITLE
TensorFlow

### DISCUSSION THREAD

**Discussion by shahbaz-prog** [AammarTufail/python_ka_chilla2023#95] [Q&A]
<img width="463" alt="image" src="https://user-images.githubusercontent.com/119630744/216069338-83620725-886e-4015-bef8-c1a0cb13c74f.png">

Can any one explain the above error and how to resolve it.

Thanks

**Comment by Muhammad-Ali-Butt** [comment]
```
fashion_mnist = tf.keras.datasets.fashion_mnist
(training_images, training_labels), (test_images, test_labels) = fashion_mnist.load_data()
```

do this in cell 2

**Reply by shahbaz-prog** [reply]
Thanks Ali
I have done it with the above code and then tried with the other, both producing the error.

**Reply by Muhammad-Ali-Butt** [reply]
shabaz bhai for now i cant figure out this one maybe try clearing output n restarting kernel and try again or do this :

i also faced a lot of errors in tensorflow and were resolved by reinstalling it maybe that would help do give it a try .
open conda then delete current env and make new one with python version 3.9(conda create --name tf_env python=3.9) then install tensorflow without specifying any version i.e. i initially downloaded tensorflow like this: pip install tensorflow<11 so then i installed it simply as pip install tensorflow . after this pip installed seaborn, plotly, openpyxl, pandas, matplotlib plus scikit-learn and thats it after this you can install opencv in the same environment in vscode using pip install opencv

**Reply by mahnoor-byte** [reply]
Also import keras in cell 1 and check if it works.  

`from tensorflow import keras`

![image](https://user-images.githubusercontent.com/121869407/216419003-a6e632d7-8931-4006-bed1-9ec147c2dabb.png)
